frontend/src/components/ui/AppFacilityFilter.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-col cols="auto">
```

Comment:
You might find it easier to wrap everything in a row otherwise if the parent doesn't have a row you end up with unparented cols. You probably will want justify="end" to keep alignment correct.

Reply:
Fixed.

Reply:
Added justify prop... in one case we want it justfy=end (on applications) and the other default justify=start.